contracts/LockedFund/LockedFund.sol

```diff
@@ -502,53 +669,125 @@ contract LockedFund is ILockedFund {
 	/**
```

Comment:
function _createVesting never returns value although expected in other functions

Reply:
Good find!

Solved in 3a11023